gravitational/stolon-app
# stolonctl running with normal user

- Init containers entrypoint moved to different image
- User id for stolon changed to 65533 to be the same in all containers

### Testing done:
1. Install cluster
2. Check all pods are working
3. Test stolonctl status is working

``` shell
sergei-dev-0:/$ kubectl exec stolon-stolonctl-946575cb8-j8wmf -- stolonctl status
NAME                                    READY   STATUS          IP              NODE            AGE     KEEPER_ID       HEALTHY         ROLE
stolon-keeper-7qvkp                     3/3     Running         100.96.68.13    10.164.0.26     13m     2c7811a0        Yes             master
stolon-keeper-hfkcv                     3/3     Running         100.96.100.15   10.164.0.28     13m     84e4817f        Yes             standby
stolon-keeper-wfcgb                     3/3     Running         100.96.64.16    10.164.0.23     13m     fd79d25a        Yes             standby
stolon-sentinel-79888945d6-6w5l2        2/2     Running         100.96.100.16   10.164.0.28     13m     N/A             N/A             N/A
stolon-sentinel-79888945d6-gvql6        2/2     Running         100.96.64.14    10.164.0.23     13m     N/A             N/A             N/A
stolon-sentinel-79888945d6-zlxpv        2/2     Running         100.96.68.12    10.164.0.26     13m     N/A             N/A             N/A

Cluster status: Healthy
```
4. Check user id for stolonctl pod

``` shell
sergei-dev-0:/$ kubectl exec stolon-stolonctl-946575cb8-j8wmf -- id
uid=65533(stolon) gid=1000(stolon) groups=1000(stolon)
```
